core/Pos/Core/ConfigPostgres.hs

```diff
@@ -0,0 +1,23 @@
+-- FIXME: Move to blockchain-importer package
+-- | Configuration of Postgres DB.
+
+{-# LANGUAGE RankNTypes #-}
```

Comment:
Very minor detail: Not needed since the corresponding Cabal file already includes `ConstraintKinds` by default.